wg-releases/triaging-issues.md

```diff
@@ -0,0 +1,64 @@
+# Electron Issues Triage
+
+Triaging issues in Electron allows maintainers to more quickly parse context and determine if a given issue is an issue that should be prioritized. It also allows for more effectively determining who has requisite context to solve a given issue.
+
+Issue triage is not an exact science, but the following general rules are helpful to keep in mind when working to triage.
+
+## Check Basics
+
+**Is this an issue at all?**
+
+If the issue is a question about a module in Electron or a request to help an opener solve a bug in their own code, the issue tracker is not currently the appropriate forum for this. As such, issues of this nature should be closed:
+
+> Thanks for reaching out!
+
+> Because we treat our issues list as the team's backlog, we close issues that are questions since they don't represent a task needing to be completed. For most questions about Electron there are a lot of options. 
+
+> Check out the [Electron community](https://github.com/electron/electron#community). There are also a bunch of helpful people in this [community forum](https://discuss.atom.io/c/electron) that should be willing to point you in the right direction.
+
+**Has the issue opener ignored the template?**
+
+If they have, that issue should be marked with the `blocked` label and more information should be requested. The following provides a typical response we may give in that case, but you may adapt it to your needs:
+
+> We require the template to be filled out on all new issues and pull requests. We do this so that we can be certain we have all the information we need to address your submission efficiently. This allows the maintainers to spend more time fixing bugs, implementing enhancements, and reviewing and merging pull requests.
+
+> We will be able to more closely look at your issue once you do so. This issue will be closed in 10 days if the above is not addressed.
+
+If the opener has filled out the template but omitted key aspects of it, like the platform or Electron version, this is also the suitable template to use. You may also add a bit of extra information to it specifying what exactly they omitted from the template and why it is important.
+
+**Apply Basic Labels** 
+
+From the information provided in the template, important labels to add include:
+
+For Bugs:
+* Operating System (e.g. `platform/macOS`, `platform/windows`, `platform/linux`)
+* Affected Release Line (e.g. `10-x-y`, `9-x-y`, etc)
+* Bug Type (`bug`, `bug/crash`, `bug/regression`)
+
+For Features
+* The `enhancement` label
+* Operating System (if applicable)
+
+There are also a series of `component` labels touching different subsystems of Electron. If for example the opener is reporting a new bug or requesting a new feature for the `menu` module, it would also be appropriate to apply the `component/menu` label. Not all subsystems have labels - you may make new ones or omit the component at your discretion.
+
```

Comment:
Maybe here we can add a line item about posting beta issues to our project board?

Reply:
the reason that i didn't is mostly because many of the items in the playbook are _already_ a good bit outdated - for example, i think it's important to set a clear expectation of when an issue will be closed should no response be provided.

We can definitely also update the playbook, but I think i'd prefer to update those items separately. I can definitely link that in for now, but i think i may prefer leaving the actual text of potential responses on this page as well 🤔

Reply:
Great points. Sounds good to me!

Reply:
All great points. Sounds good to me! We can focus on maintaining this page then